### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Create an app.rb file that serves as the console app entry-point.
- [x] Define an entry point in main.rb.

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  end
+
+  def list_people
+    puts 'No people registered yet' if @people.empty?
+    puts
+    @people.each_with_index do |person, index|
+      print "INDEX: #{index} belongs to #{person.name}, age #{person.age} is #{person.class}, with ID #{person.id}\n"
+    end
+    puts
+  end
```
Comment (Meltrust) [inline app/app.rb:53]: - [ ] Please, kindly make your interface listings exactly as in the video.  

Your Implementation | Video
:---:|:---:
image | image

That way, your interface listings will look and feel like in the video 💪

File: app/app.rb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      sleep 2 and system 'clear'
+      return
+    end
+    select_book
+    select_person
+    date = validates_date
+    rent = Rental.new(date, @book, @person)
+    @rentals << rent unless @rentals.include?(rent)
+    puts "Rental created successfully\n\n"
+  end
```
Comment (Meltrust) [inline app/app.rb:124]: - [ ] Please, kindly make the "create rental" option interface look and behave exactly as in the video:

Your Implementation | Video
:---:|:---:
image | image

That way, your rentals interface will have a better UX 💪

File: main.rb [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
-# classroom1 = Classroom.new('1A')
-# student1 = Student.new(18, name: 'John')
-# person1 = Person.new(18, name: 'John', parent_permission: true)
-# book1 = Book.new('Harry Potter', 'J.K. Rowling')
+def main
+  system 'clear'
+  puts '         ***********************'
+  puts '         * School Library v1.0 *'
+  puts '         ***********************'
+  puts
```
Comment (Meltrust) [inline main.rb:8]: - [ ] The text "Welcome to School Lybrary App" is missing from the header of the app when it starts.  Kindly add it so your app behaves as in the video! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @fabgrel10,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice code organization✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 3 above)
Comment (fabgrel10) [reply main.rb:8]: implemented

File: app/app.rb (hunk 1 above)
Comment (fabgrel10) [reply app/app.rb]: Implemented

File: app/app.rb (hunk 2 above)
Comment (fabgrel10) [reply app/app.rb:124]: Implemented

**Review by Kingstalux — APPROVED** [review]
Hi @fabgrel10,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabgrel10** [comment]
Thank you for the reviews @Meltrust and @Kingstalux!

**Pull request merged**